### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Afroz562** [community/community#40760] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi Team ,

I am trying the below command and failing the with error. please advise on this 
 aws ecr describe-images --repository-name ecs_repo \
--query 'sort_by(imageDetails,& imagePushedAt)[*].imageTags[0]' --output yaml \
| tail -n 2 | awk -F'- ' '{print $2}' | head -1  >> $GITHUB_ENV

Error: Unable to process file command 'env' successfully.
Error: Invalid format '1.0-87-2022-12-04'

**Comment by airtower-luna** [comment]
The error message indicates that what you're writing to `$GITHUB_ENV` isn't in the right format. You can find the right format [in the documentation](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#environment-files).

If you have trouble formatting it correctly, try posting what you get from that `aws` command so people have a change at figuring out what you might need to do.

**Comment by mohamedzhioua** [comment]
@Afroz562  you can check the link below to see how you can write following  the right format : "https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#environment-files"
